quickwit-cli/src/cli.yaml

```diff
@@ -379,6 +379,35 @@ subcommands:
         settings:
             - ArgRequiredElseHelp
         subcommands:
+            - add:
+                about: Adds a new source.
+                args:
+                    - index:
+                        about: ID of the target index.
+                        long: index
+                        value_name: INDEX
+                        required: true
+                    - id:
+                        about: ID of the source.
+                        long: id
+                        value_name: SOURCE ID
+                        required: true
+                    - type:
+                        about: "Type of the source. Available types are: `file` and `kafka`."
+                        long: type
+                        value_name: SOURCE TYPE
+                        required: true
+                    - params:
```

Comment:
The required structure/fields for params is unclear

Reply:
That's right. `params` is source specific. I can try to make that clearer.